examples/event_sources/src/kinesis_firehose_delivery_stream.py

```diff
@@ -1,28 +1,25 @@
-import base64
-import json
-
 from aws_lambda_powertools.utilities.data_classes import (
     KinesisFirehoseEvent,
+    KinesisFirehoseResponse,
     event_source,
 )
 from aws_lambda_powertools.utilities.typing import LambdaContext
 
 
 @event_source(data_class=KinesisFirehoseEvent)
 def lambda_handler(event: KinesisFirehoseEvent, context: LambdaContext):
-    result = []
+    result = KinesisFirehoseResponse()
 
     for record in event.records:
         # if data was delivered as json; caches loaded value
         data = record.data_as_json
 
-        processed_record = {
-            "recordId": record.record_id,
-            "data": base64.b64encode(json.dumps(data).encode("utf-8")),
-            "result": "Ok",
-        }
+        ## do all kind of stuff with data
+
```

Comment:
Add data transformation here. Don't need something too complex, but just to show an example.